### PR TITLE
the bestest new update farmhelper will stop getting checked 100%% (real (real))

### PR DESCRIPTION
**Pull request by InfiniteVelocityXVI**
fr avoids bans and ban bypass too

### CODE REVIEW

File: farmhelper-mod/src/main/java/com/jelly/farmhelper/FarmHelper.java [hunk 1]
```diff
@@ -32,6 +33,8 @@
 import java.util.jar.Manifest;
 
 
+
```
Comment (TomJuri) [inline farmhelper-mod/src/main/java/com/jelly/farmhelper/FarmHelper.java:36]: Why are there new empty lines?

File: farmhelper-mod/src/main/java/com/jelly/farmhelper/FarmHelper.java [hunk 2]
```diff
@@ -49,6 +52,9 @@ public class FarmHelper {
     public static Config config;
     public static Gson gson = new GsonBuilder().excludeFieldsWithoutExposeAnnotation().setPrettyPrinting().create();
 
+    // NickEH30 player
```
Comment (TomJuri) [inline farmhelper-mod/src/main/java/com/jelly/farmhelper/FarmHelper.java:55]: More empty lines

File: farmhelper-mod/src/main/java/com/jelly/farmhelper/FarmHelper.java [hunk 3]
```diff
@@ -75,7 +81,11 @@ public void init(FMLInitializationEvent event) {
         ClientCommandHandler.instance.registerCommand(new RewarpCommand());
         ClientCommandHandler.instance.registerCommand(new FarmHelperCommand());
 
+        //play gif
+
```
Comment (TomJuri) [inline farmhelper-mod/src/main/java/com/jelly/farmhelper/FarmHelper.java:85]: More unecessary lines

File: farmhelper-mod/src/main/java/com/jelly/farmhelper/nickeh/thirty/nick30.gif
Comment (May2Beez) [inline farmhelper-mod/src/main/java/com/jelly/farmhelper/nickeh/thirty/nick30.gif:1]: Gif not related to farming

File: farmhelper-mod/src/main/java/com/jelly/farmhelper/mixins/client/MinecraftMixin.java [hunk 4]
```diff
@@ -28,6 +30,7 @@ public class MinecraftMixin {
     //R.I.P. Fastbreak 2022-2022
     // GUESS WHO IS BACK!
 
+
```
Comment (TomJuri) [inline farmhelper-mod/src/main/java/com/jelly/farmhelper/mixins/client/MinecraftMixin.java:33]: AAAAAAAAAAAA

File: farmhelper-mod/src/main/resources/gifs/nick30.gif
Comment (May2Beez) [inline farmhelper-mod/src/main/resources/gifs/nick30.gif:1]: Why two same gifs?

File: farmhelper-mod/src/main/java/com/jelly/farmhelper/mixins/client/MinecraftMixin.java [hunk 5]
```diff
@@ -88,4 +91,12 @@ private void sendClickBlockToController(CallbackInfo ci) {
                 if (i % 3 == 0) this.thePlayer.swingItem();
             }
     }
+
+    // gif loader
+     PlayMedia playM = new PlayMedia();
+    @Inject(method= "<init>", at = @At("RETURN"))
+    private void Minecraft(GameConfiguration gameConfig,CallbackInfo info) {
```
Comment (TomJuri) [inline farmhelper-mod/src/main/java/com/jelly/farmhelper/mixins/client/MinecraftMixin.java:98]: Looks good!

**Review by TomJuri — CHANGES_REQUESTED** [review]

File: farmhelper-mod/src/main/java/com/jelly/farmhelper/FarmHelper.java
Comment (May2Beez) [inline farmhelper-mod/src/main/java/com/jelly/farmhelper/FarmHelper.java:1]: Empty lines?

File: farmhelper-mod/src/main/java/com/jelly/farmhelper/mixins/client/MinecraftMixin.java
Comment (May2Beez) [inline farmhelper-mod/src/main/java/com/jelly/farmhelper/mixins/client/MinecraftMixin.java:1]: I doubt you needed to create new object inside the mixin, you could just do this with static methods

**Review by May2Beez — CHANGES_REQUESTED** [review]
Fix the gif with something related to farm helper

File: farmhelper-mod/src/main/java/com/jelly/farmhelper/FarmHelper.java (hunk 3 above)
Comment (InfiniteVelocityXVI) [reply farmhelper-mod/src/main/java/com/jelly/farmhelper/FarmHelper.java]: shut up they are necessary

File: farmhelper-mod/src/main/java/com/jelly/farmhelper/nickeh/thirty/nick30.gif
Comment (InfiniteVelocityXVI) [reply farmhelper-mod/src/main/java/com/jelly/farmhelper/nickeh/thirty/nick30.gif:1]: ye it is

File: farmhelper-mod/src/main/java/com/jelly/farmhelper/mixins/client/MinecraftMixin.java
Comment (InfiniteVelocityXVI) [reply farmhelper-mod/src/main/java/com/jelly/farmhelper/mixins/client/MinecraftMixin.java:1]: mixin is better tho

**Review by May2Beez — CHANGES_REQUESTED** [review]
Gif is still not related to farm helper

File: farmhelper-mod/src/main/java/com/jelly/farmhelper/mixins/client/MinecraftMixin.java
Comment (May2Beez) [reply farmhelper-mod/src/main/java/com/jelly/farmhelper/mixins/client/MinecraftMixin.java:1]: Read my comment again and think about your answer

File: farmhelper-mod/src/main/java/com/jelly/farmhelper/nickeh/thirty/nick30.gif
Comment (May2Beez) [reply farmhelper-mod/src/main/java/com/jelly/farmhelper/nickeh/thirty/nick30.gif:1]: No, non fortnite streamers are not related to fh

File: farmhelper-mod/src/main/java/com/jelly/farmhelper/mixins/client/MinecraftMixin.java
Comment (InfiniteVelocityXVI) [reply farmhelper-mod/src/main/java/com/jelly/farmhelper/mixins/client/MinecraftMixin.java:1]: i prefer to stay dumb thanks 👍

**Pull request closed without merging**